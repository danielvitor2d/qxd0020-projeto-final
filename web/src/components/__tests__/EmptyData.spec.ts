import { shallowMount } from '@vue/test-utils';
import { describe, expect, it } from 'vitest';
import EmptyDataVue from '../EmptyData.vue';

describe('EmptyDataVue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(EmptyDataVue, {
      props: {
        primary: true
      }
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
