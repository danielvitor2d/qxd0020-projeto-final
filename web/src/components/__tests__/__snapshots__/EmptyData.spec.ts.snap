// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EmptyDataVue > renders correctly 1`] = `
"<div class=\\"flex flex-col items-center justify-center py-20 px-4 border-2 border-dashed border-gray-300 rounded-lg\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" class=\\"w-16 h-16 text-gray-500 mb-4\\" viewBox=\\"0 0 20 20\\" fill=\\"currentColor\\">
    <circle cx=\\"10\\" cy=\\"10\\" r=\\"8\\"></circle>
    <path d=\\"M10 14.5a.5.5 0 01-.5-.5v-5a.5.5 0 011 0v5a.5.5 0 01-.5.5z\\" fill=\\"#fff\\"></path>
  </svg>
  <p class=\\"text-gray-500 text-lg font-semibold mb-2\\">Dados vazios</p>
  <p class=\\"text-gray-400\\">Não há informações disponíveis no momento.</p>
</div>"
`;
